earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -148,12 +158,8 @@ install_or_upgrade_asset_root()
         chown -R "$GEFUSIONUSER:$GEGROUP" "$ASSET_ROOT"
     else
         # upgrade asset root -- if this is a master
-        if [ "$IS_SLAVE" = false ]; then
-            # TODO: Verify this logic -- this is what is defined in the
-            # installer documentation, but needs confirmation
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gegroup_existed NEW_GEFUSIONUSER
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gefusionuser_existed NEW_GEFUSIONUSER
-            if [ "$NEW_GEGROUP" = true ] || [ "$NEW_GEFUSIONUSER" = true ]; then
+        if [ "$IS_SLAVE" = "false" ]; then
+            if [ "$NEW_INSTALL" = "false" ] ; then
                 NOCHOWN=""
                 UPGRADE_MESSAGE="The upgrade will fix permissions for the asset root and source volume. This may take a while."
```

Comment:
This talks about "upgrade", but not with the same meaning as rpm upgrade.  In the case of rpm's, we have nailed down the gegroup and users for the life of the rpm installs.  Hence, "upgrading", but from one rpm to a newer one, should I believe normally be done with --nochown, and the original NEW_INSTALL test a few lines about should actually be if true, as the full chown should be done if it is a new install and yet there is a prior asset root.  I think this misunderstanding is the real issue, and the upgrade-migration test then does actually become unnecessary.